tests/test_player.py

```diff
@@ -0,0 +1,105 @@
+from openmedia.player import mixer
+
+import unittest
+import os
+
+class TestPlayerFunctionality(unittest.TestCase):
+
+    def setUp(self):
+        track_names = ["1.mp3",
+                       "2.mp3",
+                       "3.mp3"]
+        module_folder = os.path.dirname(__file__)
+        self.test_folder = os.path.join(module_folder, "test_tracks")
+        self.track_paths = [os.path.join(self.test_folder, name) \
+                            for name in track_names]
+        mixer.init(self.track_paths)
+
+    def test_init(self):        
+        self.assertTrue(mixer.track_count == 3)
+
+    def test_play(self):
+        mixer.play()
+        self.assertTrue(mixer.is_playing())
+        self.assertFalse(mixer.is_paused)
+        self.assertFalse(mixer.is_stopped)
+        self.assertEqual(mixer.curr_track_index, 0)
+        self.assertEqual(mixer.current_track.get_path(),
+                         self.track_paths[0])
+
+    def test_pause(self):
+        mixer.play()
+        mixer.pause()
+        self.assertFalse(mixer.is_playing())
+        self.assertTrue(mixer.is_paused)
+        self.assertFalse(mixer.is_stopped)
+        self.assertEqual(mixer.curr_track_index, 0)
+        self.assertEqual(mixer.current_track.get_path(),
+                         self.track_paths[0])
+
+    def test_stop(self):
+        mixer.play()
+        mixer.stop()
+        self.assertFalse(mixer.is_playing())
+        self.assertFalse(mixer.is_paused)
+        self.assertTrue(mixer.is_stopped)
+        self.assertEqual(mixer.curr_track_index, 0)
+        self.assertEqual(mixer.current_track.get_path(),
+                         self.track_paths[0])
```

Comment:
Well, the `0` in `self.track_paths[0]` has nothing to do with `mixer.curr_track_index`. Just because in this case they are equal and inside mixer there is a copy of `track_paths`, there is no use to change it from `0` to `mixer.curr_track_index`. I can change it if you really want to, but there is no use.